src/collective/casestudy/profiles/default/workflows/provider_workflow/definition.xml

```diff
@@ -0,0 +1,345 @@
+    </permission-map>
+  </state>
+  <state state_id="published"
+         title="Published"
+         i18n:attributes="title"
+  >
+    <description i18n:translate="">
+    Visible to everyone, editable by the owner and editor.
+  </description>
+    <exit-transition transition_id="retract" />
+    <exit-transition transition_id="reject" />
+    <permission-map acquired="False"
+                    name="Access contents information"
+    >
+      <permission-role>Anonymous</permission-role>
+    </permission-map>
+    <permission-map acquired="False"
+                    name="Modify portal content"
+    >
+      <permission-role>Manager</permission-role>
+      <permission-role>Owner</permission-role>
+      <permission-role>Editor</permission-role>
+      <permission-role>Site Administrator</permission-role>
+    </permission-map>
+    <permission-map acquired="False"
+                    name="View"
+    >
+      <permission-role>Anonymous</permission-role>
+    </permission-map>
+    <permission-map acquired="False"
+                    name="collective.casestudy: View Contact Information"
+    >
+      <permission-role>Manager</permission-role>
+      <permission-role>Owner</permission-role>
+      <permission-role>Editor</permission-role>
+      <permission-role>Reviewer</permission-role>
+      <permission-role>Site Administrator</permission-role>
+    </permission-map>
+    <permission-map acquired="False"
+                    name="collective.casestudy: View Organization Information"
+    >
+      <permission-role>Manager</permission-role>
+      <permission-role>Owner</permission-role>
+      <permission-role>Editor</permission-role>
+      <permission-role>Reviewer</permission-role>
+      <permission-role>Site Administrator</permission-role>
+    </permission-map>
+  </state>
+
+  <transition after_script=""
+              before_script=""
+              new_state="published"
+              title="Reviewer publishes content"
+              transition_id="publish"
+              trigger="USER"
+              i18n:attributes="title"
+  >
+    <description i18n:translate="">
+    Publishing the item makes it visible to other users.
+  </description>
+    <action category="workflow"
+            url="%(content_url)s/content_status_modify?workflow_action=publish"
+            i18n:translate=""
+    >Publish</action>
+    <guard>
+      <guard-permission>Review portal content</guard-permission>
+    </guard>
+  </transition>
+  <transition after_script=""
+              before_script=""
+              new_state="private"
+              title="Reviewer sends content back for re-drafting"
+              transition_id="reject"
+              trigger="USER"
+              i18n:attributes="title"
+  >
+    <description i18n:translate="">
+    Sending the item back will return the item to the original author instead of publishing it. You should preferably include a reason for why it was not published.
```

Comment:
```suggestion
    Sending the item back will return the item to the original author instead of publishing it. Include your reason for not publishing it.
```